graphql/schema/types/patch.graphql

```diff
@@ -74,6 +74,7 @@ type Patch {
   project: PatchProject
```

Comment:
Can you see if it's possible to delete this field? It seems like it's unused by the frontend and it could be confusing to have two different fields with similar names. No worries if it's not possible to delete it though!